build.gradle

```diff
@@ -234,12 +259,17 @@ tasks.register("stop") {
 
 clean {
 	dependsOn ":cleanPrepareHotfixes"
+	dependsOn ":cleanConfigureWebserver"
 }
 
 dockerDeploy {
 	if (useLiferay) {
 		dependsOn ":prepareHotfixes"
 	}
+
+	if (useWebserver) {
+		dependsOn ":configureWebserver"
+	}
```

Comment:
So `dockerDeploy` is actually specific to the building of the Liferay image and not necessarily related to the web server container build. I think it would be better to have `composeBuild` depend on the web server task instead.